server/routes/ad.ts

```diff
@@ -715,6 +719,34 @@ export default class AdService {
                   },
                 },
               },
+              ...(entityName && entityValue
+                ? [
+                    {
+                      nested: {
+                        path: 'entity',
+                        query: {
+                          term: {
+                            'entity.name': {
```

Comment:
User [`ENTITY_NAME_PATH_FIELD`](ENTITY_NAME_PATH_FIELD) ?
Same question for `'entity.value'`